src/evergreen/api.py

```diff
@@ -967,14 +968,17 @@ def _execute_patch_file_command(
             command = f"{command} --author {author}"
 
         process = subprocess.run(command, shell=True, capture_output=True)
+        output = str(process.stderr)
 
-        match = re.search(EVERGREEN_URL_REGEX, str(process.stderr))
-        if match is None:
+        url_match = EVERGREEN_URL_REGEX.search(output)
+        id_match = EVERGREEN_PATCH_ID_REGEX.search(output)
+
+        if url_match is None or id_match is None:
             raise Exception(
-                f"Unable to parse URL from command output: {str(process.stderr)}. \nExecuted command: {command}"
+                f"Unable to parse URL or ID from command output: {str(process.stderr)}. \nExecuted command: {command}"
```

Comment:
instead of str(process.stderr), you can put "output" since you have it readily available on line 971.